bin/doc-test

```diff
@@ -7,12 +7,14 @@ program
   .version(pkg.version)
   .option('-f, --file [pathToFile]','File needs to be tested', 'README.md')
   .option('-d, --debug [true | false]','Retain test file after execution')
-  .action(async (command) => {
-    let { file, debug } = command;
+  .option('--inspect|--inspect-brk [true | portNumber]','Debug the test file.')
+  .action(async () => {
+    let { file, debug, inspectBrk } = program;
+    let inspect =/true|1|\d/.test(inspectBrk) ? inspectBrk : false;
     try {
-      await markDownTest(file, /true|1/.test(debug));
+      await markDownTest(file, /true|1/.test(debug), inspect);
     } catch (err) {
-      console.log(`Not all tests passed: \n${err}`);
+      console.log(`Not all tests passed: ${err}`);
```

Comment:
we should likely also print out the error

Reply:
We do print the error
```
node --inspect-brk bin/doc-test
Debugger listening on ws://127.0.0.1:9229/e7b1bb0d-1ab7-400b-97c5-c63e115c531d
For help see https://nodejs.org/en/docs/inspector
Debugger attached.
TAP version 13
ok 1 README.md > add(4, 5)
ok 2 README.md > add(4, 5)
ok 3 README.md > add(2, 3)
not ok 4 README.md > subtract(numbers.a, numbers.b)
  ---
  message: "Died on test #1     at Object.<anonymous> (/Users/snrai/Projects/doc-tester/test.js:23:3): c is not defined"
  severity: failed
  actual: null
  expected: undefined
  stack: ReferenceError: c is not defined
    at exports.subtract (/Users/snrai/Projects/doc-tester/add.js:11:3)
    at Object.<anonymous> (/Users/snrai/Projects/doc-tester/test.js:24:36)
  ...
ok 5 README.md > add( {  a: 4}.a, 4)
1..5
# pass 4
# skip 0
# todo 0
# fail 1
Not all tests passed: Test exited with code 1
```